drivers/i2c/Kconfig

```diff
@@ -13,6 +13,17 @@ menuconfig I2C
 
 if I2C
 
+config I2C_FALLBACK_ADDRESSES
+	int "Enable I2C fallback addresses"
```

Comment:
"Enable" is redundant.

Also its prompt and description are not accurate. This config is really **the number of maximum supported** I2C fallback addresses.